K2Bridge/Visitors/MetricAggregationVisitor.cs

```diff
@@ -57,5 +57,22 @@ public void Visit(SumAggregation sumAggregation)
 
             sumAggregation.KustoQL = $"['{sumAggregation.Key}']={KustoQLOperators.Sum}({sumAggregation.Field})";
         }
+
+        /// <inheritdoc/>
+        public void Visit(PercentileAggregation percentileAggregation)
+        {
+            Ensure.IsNotNull(percentileAggregation, nameof(percentileAggregation));
+            EnsureClause.StringIsNotNullOrEmpty(percentileAggregation.Field, percentileAggregation.Field, ExceptionMessage);
+
+            /// Median is Percentile(fieldName, 50)
+            percentileAggregation.KustoQL = string.Empty;
+
+            foreach (double percent in percentileAggregation.Percents)
+            {
+                percentileAggregation.KustoQL += $"['{percentileAggregation.Key}%{percent:0.0}']={KustoQLOperators.Percentile}({percentileAggregation.Field}, {percentileAggregation.Percents[0]}),";
+            }
+
+            percentileAggregation.KustoQL = percentileAggregation.KustoQL.TrimEnd(',');
```

Comment:
@nebrass I think my previous comment got lost somehow...
Using multiple percentile function is less preformant than a single precenrile**s**. Did you consider it?

Reply:
Fixed; now percentiles_array is used